src/components/pages/LoginPage.tsx

```diff
@@ -1,57 +1,27 @@
-import React, { useEffect, useState } from 'react';
-import { useAtomValue, useSetAtom } from 'jotai';
-import { playerIdAtom } from '../../atoms/PlayerAtoms';
-import { gameScreenAtom, nicknameAtom } from '../../atoms/GameAtoms';
+import React, { ChangeEvent, useCallback } from 'react';
+import { useAtom, useSetAtom } from 'jotai';
+import { playerInfoAtom } from '../../atoms/PlayerAtoms';
+import { gameScreenAtom } from '../../atoms/GameAtoms';
 import { GameScreen } from '../../types/game';
+import useGame from '../../hooks/useGame';
 
 const LoginPage = () => {
-  const playerId = useAtomValue(playerIdAtom);
-  const serverNickname = useAtomValue(nicknameAtom);
-  const [customNickname, setCustomNickname] = useState<string>('');
-  const setNickname = useSetAtom(nicknameAtom);
-  const [error, setError] = useState<string | null>(null);
-  const [isLoading, setIsLoading] = useState(false);
+  const [player, setPlayer] = useAtom(playerInfoAtom);
   const setGameScreen = useSetAtom(gameScreenAtom);
+  const { registerPlayerQuery } = useGame();
 
-  // 게임 시작 함수
-  const handleGameStart = async () => {
-    if (!playerId) return;
-    const nickname = customNickname.trim() || serverNickname; // 입력값이 없으면 서버 닉네임 사용
+  const handleRegisterPlayer = useCallback(async () => {
+    const userId = await registerPlayerQuery(player);
+    setGameScreen(GameScreen.MATCHING);
```

Comment:
서버합 맞추다가 발견했어요!

```suggestion
    setGameScreen(GameScreen.HOME);
```

LOADING -> LOGIN -> HOME 순서인데 확인 부탁 드려요!

****

그리고 
LOGIN : 입장하기
HOME: 빠른 시작

이렇게 CTA 명 수정하면 어떨까요? (연속으로 시작하기 > 시작하기 라 어색하게 느껴집니다)